test/testdata/packager/lsp_features/__package.rb

```diff
@@ -6,7 +6,7 @@ class Simpsons < PackageSpec
     # go-to-def on a reference to Bart within the package goes here, but go-to-def on Bart in `import Bart` goes to the
     # package.
     import Bart
-    #      ^^^^ symbol-search: "Bart", name = "Simpsons::Bart", container = "Simpsons"
+    #      ^^^^ symbol-search: "Bart", name = "Bart", container = "Simpsons"
```

Comment:
It looks like you've regressed symbol search in `--stripe-packages` mode. We'll soon be using this mode internally at Stripe. Can we restore the container being part of the name in this case?

Reply:
I think this is the desired behavior and https://github.com/sorbet/sorbet/pull/3475 regressed it (this is going back to state before that PR). I could be wrong though